src/accessors/IModify.ts

```diff
@@ -0,0 +1,241 @@
+import { IMessage, IMessageAttachment } from '../messages';
+import { IRoom, RoomType } from '../rooms';
+import { IUser } from '../users';
+
+export interface IModify {
+    update(): IModifyUpdate;
```

Comment:
`getUpdater()` might be better wording to get the interface for updating objects.